lnbits/static/js/components/payment-list.js

```diff
@@ -375,9 +382,9 @@ Vue.component('payment-list', {
                           :payment="props.row"
                         ></lnbits-payment-details>
                       </div>
-                      <div v-else-if="props.row.isOut && props.row.pending">
-                        <q-icon name="settings_ethernet" color="grey"></q-icon>
-                        <span v-text="$t('outgoing_payment_pending')"></span>
+                      <div v-else-if="props.row.isFailed">
```

Comment:
The color of the icons is different for failed payments.

![image](https://github.com/lnbits/lnbits/assets/2951406/82763563-c979-44d0-9b45-e5d7b16fa507)

Maybe a [different icon](https://fonts.google.com/icons?icon.set=Material+Icons&selected=Material+Icons+Outlined:priority_high:&icon.query=error&icon.size=24&icon.color=%23e8eaed) should be used for failed payments.

Reply:
`Failed` payments should not be retuned unless explicitly required so.
For example `BlueWallet` will list them as successful.


![image](https://github.com/lnbits/lnbits/assets/2951406/24950c3e-7fda-4ba1-bfd4-3e2f6f61fe9e)